docs/tutorials/kubernetes-basics/index.html

```diff
@@ -26,6 +26,7 @@ <h2>Kubernetes Basics</h2>
         <li>Debug the containerized application</li>
         </ul>
         <p>The tutorials use Katacoda to run a virtual terminal in your web browser that runs Minikube, a small-scale local deployment of Kubernetes that can run anywhere. There's no need to install any software or configure anything; each interactive tutorial runs directly out of your web browser itself.</p>
+	<p>Note that you may have to disable or customize the AdBlocker (if any) in your browser to view the modules.</p>
```

Comment:
Capitalizing "AdBlocker" might lead users to think that we're referring to a specific ad blocker. Let's keep the reference more generic, and maybe rephrase a little, to something like:
"You might need to disable or customize any ad blockers that you've added to your browser." (Don't need the parenthetical "if any" ...)
Thanks!

Reply:
ok